cmake/std/sems/PullRequestPython3TestingEnv.sh

```diff
@@ -37,12 +37,9 @@ module load atdm-ninja_fortran/1.7.2
 # to one that has the proper sym-links from python -> python3
 module unload sems-python
 # module load sierra-python/3.6.3 - permissions do not allow this, but the execs are ok
-PATH=/projects/sierra/linux_rh7/install/Python/3.6..3/bin:${PATH}
+export PATH=/projects/sierra/linux_rh7/install/Python/3.6..3/bin:${PATH}
```

Comment:
@prwolfe Did you mean to have two dots here between 6 and 3?

Reply:
No I did not - this will fail.

Reply:
I wonder how I got it to run on my blade? Must have had both in place.